tutorials/clinical/10_ieeg_localize.py

```diff
@@ -0,0 +1,505 @@
+# shape and size of brain areas, we need to fix the alignment of the brains.
+# The plot below shows that they are not yet aligned.
+
+# load the freesurfer average brain
+template_brain = nib.load(
+    op.join(subjects_dir, 'fsaverage', 'mri', 'brain.mgz'))
+
+plot_overlay(template_brain, subject_brain,
+             'Alignment with fsaverage before Affine Registration')
+
+###############################################################################
+# Now, we'll register the affine of the subject's brain to the template brain.
+# This aligns the two brains, preparing the subject's brain to be warped
+# to the template.
+
+# normalize intensities
+mri_to = template_brain.get_fdata().copy()
+mri_to /= mri_to.max()
+mri_from = subject_brain.get_fdata().copy()
+mri_from /= mri_from.max()
+
+# downsample for speed
+zooms = (5, 5, 5)
+mri_to, affine_to = reslice(
+    mri_to, affine=template_brain.affine,
+    zooms=template_brain.header.get_zooms()[:3], new_zooms=zooms)
+mri_from, affine_from = reslice(
+    mri_from, affine=subject_brain.affine,
+    zooms=subject_brain.header.get_zooms()[:3], new_zooms=zooms)
+
+reg_affine = affine_registration(
+    moving=mri_from,
+    static=mri_to,
+    moving_affine=affine_from,
+    static_affine=affine_to,
+    nbins=32,
+    metric='MI',
+    pipeline=[center_of_mass, translation, rigid, affine],
+    level_iters=[100, 100, 10],
+    sigmas=[3.0, 1.0, 0.0],
+    factors=[4, 2, 1])[1]
+
+# Apply the transform to the subject brain to plot it
+subject_brain_aligned = resample(moving=subject_brain.get_fdata(),
+                                 static=template_brain.get_fdata(),
+                                 moving_affine=subject_brain.affine,
+                                 static_affine=template_brain.affine,
+                                 between_affine=reg_affine)
+plot_overlay(template_brain, subject_brain_aligned,
+             'Alignment with fsaverage after Affine Registration')
+
+###############################################################################
+# Next, we'll compute the symmetric diffeomorphic registration. This accounts
+# for differences in the shape and size of the subject's brain areas
+# compared to the template brain.
+#
+# This takes several minutes, so it won't actually be executed in the
+# tutorial, instead, we'll just use pre-computed results for convenience.
+#
+# .. code-block:: python
+#
+#     from dipy.align.metrics import CCMetric
+#     from dipy.align.imwarp import SymmetricDiffeomorphicRegistration
+#     # Compute registration
+#     sdr = SymmetricDiffeomorphicRegistration(
+#         metric=CCMetric(3), level_iters=[10, 10, 5])
+#     mapping = sdr.optimize(static=template_brain.get_fdata(),
+#                            moving=subject_brain.get_fdata(),
+#                            static_grid2world=template_brain.affine,
+#                            moving_grid2world=subject_brain.affine,
+#                            prealign=reg_affine)
```

Comment:
Ahh so one difference is that this is computed at the native resolution rather than the reduced space. So maybe we need an API like `zooms=dict(affine=(5, 5, 5), sdr=(1, 1, 1))` in the `mne.morph.align_volumes` function.

Reply:
Yeah, that was the gist of my comment above, sorry should have read this one first.